app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.MyMovieInfo">
+        <activity android:name=".MovieDetailsActivity" />
```

Comment:
remove <activity android:name=".MovieDetailsActivity" />
and change
<activity android:name=".MainActivity"> -> <activity android:name=". MovieDetailsActivity">

Reply:
Done